client/app/templates/dashboard.hbs

```diff
@@ -9,11 +9,11 @@
 
 <div id='freewall' class='free-wall'>
         {{#each sortableObjectList as |item|}}
-            {{place-holder options=addableList addChart=(action "addChart") removeChart=(action 'removeChart') item=item widgets=widgets wall=wall refreshWall=(action 'refreshWall') dashboardSaveWidget=(action 'dashboardSaveWidget')}}
+            {{place-holder name=item.name options=addableList addChart=(action "addChart") removeChart=(action 'removeChart') item=item widgets=widgets wall=wall refreshWall=(action 'refreshWall') dashboardSaveWidget=(action 'dashboardSaveWidget')}}
```

Comment:
We pass in item to the component already, so we could just access it from that dict inside the component; at some point we should refactor this component to have to take less args as well.